README.md

```diff
@@ -25,3 +25,34 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+# Runtime Analysis Answer:
+
+The time complexity of my divideAndConquerSum() is $\Theta(log(n))$.
+At the beginning of my algorithm, I check to see if the length of the
+array is 0 or 1. If the array is either of these lengths, then the execution
+time will be constant in the best case scenario. However, in the scenerio
+where the length is greater than 1, the complexity increases.
+The algorithm will always divide the array up into thirds and process
+its own portion of the array until it reaches the base cases where the 
+sub-array size is 0 or 1, in which it is added in a constant time:
+
+$T(n) = 3T(n/3) + \Theta(1)$
```

Comment:
Can you show how you solved this please? I don't get $\log n$ for this recurrence relation.

Reply:
I now see the error in my solution after some research. I am working on an updated version with the correct solution and how I solved it.